### PR TITLE
KAB-179 queue drop workspace

### PR DESCRIPTION
**Pull request by tomasfejfar**
Jira: KAB-179
KBC: N/A

Before asking for review make sure that:

## Checklist

- [x] New client method(s) has tests
- [x] Apiary file is updated

## Release

  - [x] I gave the PR a proper label:
    * major (BC break)
    * minor (new feature)
    * patch (backwards compatible fix)
    * no release (just test changes)

### CODE REVIEW

**Comment by tomasfejfar** [comment]
Ještě jedna věc tam chyběla, snad poslední ;)

File: src/Keboola/StorageApi/Client.php [hunk 1]
```diff
@@ -2866,11 +2866,12 @@ public function apiPutJson(string $url, array $data = [], bool $handleAsyncTask
...
     /**
      * @param string $url
+     * @param bool $handleAsyncTask
      * @return mixed|string
      */
-    public function apiDelete($url)
+    public function apiDelete($url, bool $handleAsyncTask = true)
     {
-        return $this->request('DELETE', $url);
+        return $this->request('DELETE', $url, [], null, $handleAsyncTask);
```
Comment (romanbracinik) [inline src/Keboola/StorageApi/Client.php:2874]:
```suggestion
        return $this->request('DELETE', $url, handleAsyncTask: $handleAsyncTask);
```

Comment (romanbracinik) [reply src/Keboola/StorageApi/Client.php:2874]: uz mame  :)
```
"require": {
		"php": ">=8.1",
```

**Review by romanbracinik — APPROVED** [review]
ok, pridal som jednu poznamku ale funguje to.

File: src/Keboola/StorageApi/Client.php (hunk 1 above)
Comment (tomasfejfar) [reply src/Keboola/StorageApi/Client.php]: ❤️

**Review by zajca — APPROVED** [review]
sry nějak sem to minul :/

**Pull request merged**